### PR TITLE
Cover Art Label Local

### PR DESCRIPTION
**Pull request by fatihemreyildiz**
As we talked on the other local PR, this is the first try of coverartlabel. I tried to move the WCoverArtMenu to the constructor. If it is okay I want to move on it, then have a final on here, then I can open another Separate PR for just coverartlayout.

What do you think generally?

### CODE REVIEW

File: src/widget/wcoverartlabel.h [hunk 1]
```diff
@@ -16,7 +16,8 @@ class CoverInfoRelative;
 class WCoverArtLabel : public QLabel {
     Q_OBJECT
   public:
-    explicit WCoverArtLabel(QWidget* parent = nullptr);
+    explicit WCoverArtLabel(QWidget* parent = nullptr, WCoverArtMenu* pCoverMenu = nullptr);
```
Comment (daschuer) [inline src/widget/wcoverartlabel.h]: Can this become: 
```suggestion
    explicit WCoverArtLabel(QWidget* parent = nullptr, QMenu* pCoverMenu = nullptr);
```
Or is there any dependency to the WCoverArtMenu

Comment (daschuer) [reply src/widget/wcoverartlabel.h]: Ah, it is correct, we need this for the connection

Comment (daschuer) [reply src/widget/wcoverartlabel.h]: The connection only forward signals. Is it possible to connect them directly to the menu and remove all dependency here?

Comment (fatihemreyildiz) [reply src/widget/wcoverartlabel.h]: Only if we don't use setCoverArt from WCoverArtMenu we can remove the dependency. According to my tests, I didn't notice anything with/without that method.

**Pull request closed without merging**